From 7a9050ddc0a07ef673a58c8c55a9797adb49b682 Mon Sep 17 00:00:00 2001
From: Kai Liu <kai.liu@suse.com>
Date: Thu, 19 May 2022 09:42:37 +0800
Subject: [PATCH] Revert "Linux 5.10.117"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 7686a5c2a8d398196259b1bf3fa369a4fd6bcd6f.

Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8f611d79d5e1..c999de1b6a6b 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 117
+SUBLEVEL = 116
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.1

